University-advisor-web/Controllers/SettingsController.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using University_advisor_web.Models;
+
+namespace University_advisor_web.Controllers
+{
+    public class SettingsController : Controller
+    {
+        public IActionResult Index()
+        {
+            var model = new UserModel();
+            return View(model);
+        }
+
+        [HttpPost]
+        public IActionResult ChangePassword(UserModel model)
+        {
+            model.Username = "test";
+            model.Password = "password";
```

Comment:
What is the purpose of these hard coded values? I think you used them only for testing and they are not needed anymore.

Reply:
I will leave them for now, as there are no fucntional log in at the moment.